test/spec/openapi/refs.js

```diff
@@ -2,7 +2,7 @@
 
 const { test } = require('tap')
 const Fastify = require('fastify')
-const Swagger = require('swagger-parser')
+const Swagger = require('@apidevtools/swagger-parser')
```

Comment:
Is this change needed?

`@apidevtools/swagger-parser` does not support Node 10 (reading their CI pipeline)

Reply:
This is needed because the old swagger-parser seems to require the description field on headers when using `allOf`.
This was the error that was popping:
```
Data does not match any schemas from 'oneOf' at #/paths//url1/get/parameters/1
    Expected type string but found type undefined at #/paths//url1/get/parameters/1/description     Missing required
  property: $ref at #/paths//url1/get/parameters/1   Data does not match any schemas from 'oneOf' at
  #/paths//url1/get/parameters/0     Expected type string but found type undefined at
  #/paths//url1/get/parameters/0/description     Missing required property: $ref at #/paths//url1/get/parameters/0
  JSON_OBJECT_VALIDATION_FAILED
```